packages/server/src/lib/email.js

```diff
@@ -33,16 +35,38 @@ async function deliverEmail({
     });
 }
 
+function buildBaseUrlSafe() {
+    const baseUrl = new URL(process.env.WEBSITE_DOMAIN);
+    baseUrl.searchParams.set('utm_source', 'subscription');
```

Comment:
Just for driving towards consistency
```suggestion
    baseUrl.searchParams.set('utm_source', 'usdr-grants');
```